apps/server/src/modules/room-membership/service/room-membership.service.ts

```diff
@@ -103,6 +103,26 @@ export class RoomMembershipService {
 		await this.handleGuestRoleRemoval(userIds, roomMembership.schoolId);
 	}
 
+	public async changeRoleOfRoomMembers(roomId: EntityId, userIds: EntityId[], roleName: RoleName): Promise<void> {
+		const roomMembership = await this.roomMembershipRepo.findByRoomId(roomId);
+		if (roomMembership === null) {
+			throw new BadRequestException('Room member not found');
```

Comment:
I'd rather say "Room member list not found." or "Room membership not found." ... and maybe add "for this room".

Reply:
We would have to implement a seperate repo function for it, since in getRoomMembershipAuthorizable its used as a trigger to create the membership if it doesnt yet exist...

Reply:
which also means that in theory this error can never occur, since right before this function is called we get the authorizable in order to authorize the request...